www/entries.js

```diff
@@ -10,7 +10,7 @@ function feedback_mode() {
         elts[i].addEventListener('click', function() {
             var pw = window.localStorage["pw"];
             submit_status(parseInt(this.parentNode.dataset.id), this.className, pw);
-            this.parentNode.remove();
+            location.reload();
```

Comment:
I assume this is fast enough to not be annoying?

Reply:
> I assume this is fast enough to not be annoying?

I think it should be. Currently there are a ton of entries on the feedback page and it loads pretty fast.